docs/CLI.md

```diff
@@ -1,235 +1,3 @@
-Bcoin ships with bcoin-cli as its default HTTP client for command line access.
+Bcoin ships with [bclient](https://github.com/bcoin-org/bclient) as its default HTTP client for command line access.
 
-## Configuration
-
-Examples:
-
-``` bash
-$ export BCOIN_API_KEY=hunter2
-$ export BCOIN_NETWORK=main
-$ export BCOIN_URI=http://localhost:8332
-$ bcoin cli info
-```
-
-``` bash
-$ bcoin cli info --api-key=hunter2 --uri=http://localhost
-```
-
-``` bash
-$ echo 'api-key: hunter2' > ~/cli.conf
-$ bcoin cli info --config=~/cli.conf
-```
-
-## Examples
```

Comment:
Could also be useful to include a few of the top used bcoin-cli commands for examples.